stream-chat-android-core/src/main/java/io/getstream/chat/android/models/streamcdn/image/StreamCdnResizeImageQueryParameterKeys.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.models.streamcdn.image
+
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+/**
+ * Contains query parameter keys used to resize Stream CDN hosted images.
+ */
+@InternalStreamChatApi
+public object StreamCdnResizeImageQueryParameterKeys {
```

Comment:
Do we need this class outside of `stream-chat-android-client`?? :thinking:

Reply:
We don't really, I just wanted to keep structure consistency.

If you want it transferred to the client module and made internal I can do that. :smile: